test/views/components/ConfirmationPanelSpec.scala

```diff
@@ -55,20 +55,14 @@ class ConfirmationPanelSpec extends BaseSpec with ViewFns with GuiceOneAppPerSui
     val englishBodyText2: String = "Second line in body"
     val welshBodyText2: String = "Welsh, Second line in body"
```

Comment:
Are the Welsh string definitions still required?

Reply:
removed